src/main/java/fr/foodlimit/api/Application.java

```diff
@@ -11,6 +11,8 @@
 @SpringBootApplication
 public class Application {
 
+  private String test;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "test" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)